test/validation/api/dma/dma.c

```diff
@@ -1000,7 +1063,7 @@ static void test_dma_addr_to_addr_sync(void)
 
 static void test_dma_addr_to_addr_sync_mtrs(void)
 {
-	test_dma_addr_to_addr_trs(ODP_DMA_COMPL_SYNC, 2, 0, 0);
+	test_dma_addr_to_addr_trs(ODP_DMA_COMPL_SYNC, global.dma_capa.max_transfers * 2, 0, 0);
```

Comment:
Perhaps it would be good to add a comment here to explain why a larger transfer count is used.